### ISSUE TITLE
Request: 32:9 stereo movie playback

### ISSUE THREAD

**Issue by NoUserNameForYou** [fholger/farcry_vrmod#41]
Hello @fholger 
I can't thank you enough for your labor. You gave a new life to this old pulp gem and I'm enjoying it! TBH after Alyx, it found its way into my Top 5 vr fps. 

I have 3d sbs videos, you said 1.0 was feature complete but can't you make an exception? Pretty please? I think we should strive for perfection since the rest of the game, even cutscenes can be viewed in 3d and the movies can not.

Am I allowed to upload the 3d bik files here so you can check if it's not too much labor for you? Thank you in advance.

**Comment by NoUserNameForYou** [comment]
Since the maindev doesn't answer whether it's possible or not, I'll leave the bik files here if someone tries to make a pull request.

https://gofile.io/d/uFbbId
https://gofile.io/d/o0qqX3
https://gofile.io/d/9TWOs8
https://gofile.io/d/QnWGll

I have the unbinked mkvs, I can share if needed.

**Comment by devsnd** [comment]
I took a look how this could be done, but it's a bit tricky.

From what I could gather, the videos in the game a in-game-entities, which are uploading the video-frame as texture as part of the game-logic.. Those are updated once per game tick.

However, we render each game state twice, once per eye from a different perspective. During rendering the game world is usually not updated. However, for the 32:9 video to be projected correctly, we'd need to crop and stretch the video differently for each eye, which would need to happen during rendering, but also requires the video-update logic to run.

So long story short, it's feasible, but it would require 2 hacks. The video-player must know somehow for which eye they should update now and secondly we must run the update-logic for all video elements during the render-stage.

**Comment by NoUserNameForYou** [comment]
Thanks for the info. How about 16:9 half sbs or frame interleaved vids? Would they be easier to implement?
